src/main/scala/com/microsoft/hyperspace/index/zordercovering/ZOrderFilterIndexRule.scala

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
Just curious, what's the difference between these rules and rules for hash bucketed covering index? If they are the same, can't we reuse the code?